cinema/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from cinema.views import get_or_create_movie, get_update_or_delete_movie
+
+app_name = "cinema"
+
+urlpatterns = [
+    path("cinema_movies/", get_or_create_movie, name="cinema_movies"),
+    path("cinema_movies/<int:pk>", get_update_or_delete_movie, name="cinema_movie"),
```

Comment:
here path can actually be called `path("movies/"..` In your project url you already have path /api/cinema/ so it is fine. I meant to change alias name for url the same as you changed names for function views